tests/integration/bond_test.py

```diff
@@ -227,19 +227,13 @@ def test_set_bond_mac_address(eth1_up):
         libnmstate.apply(bond_state)
 
         current_state = statelib.show_only(('bond99', 'eth1'))
-        bond99_cur_state = current_state[INTERFACES][0]
-        eth1_cur_state = current_state[INTERFACES][1]
-        assert bond99_cur_state[Interface.MAC] == MAC0.upper()
-        assert bond99_cur_state[Interface.MAC] == eth1_cur_state[Interface.MAC]
+        assertMacAddress(current_state, MAC0.upper())
```

Comment:
The `upper` should be done in `assertMacAddress()`.

Reply:
Did it in there just so that expected_mac is consistent when it comes to comparing MACs.
Still think that upper should be in assertMacAddress() ?